commit 70509f9b4807295b2b4b43bffe110580fc0381ef
Author: Noah Goldstein <goldstein.w.n@gmail.com>
Date:   Sat Feb 12 00:45:00 2022 -0600

    x86: Set .text section in memset-vec-unaligned-erms
    
    commit 3d9f171bfb5325bd5f427e9fc386453358c6e840
    Author: H.J. Lu <hjl.tools@gmail.com>
    Date:   Mon Feb 7 05:55:15 2022 -0800
    
        x86-64: Optimize bzero
    
    Remove setting the .text section for the code. This commit
    adds that back.
    
    (cherry picked from commit 7912236f4a597deb092650ca79f33504ddb4af28)

diff --git a/sysdeps/x86_64/multiarch/memset-vec-unaligned-erms.S b/sysdeps/x86_64/multiarch/memset-vec-unaligned-erms.S
index 39a096a594ccb5b6..d9c577fb5ff9700f 100644
--- a/sysdeps/x86_64/multiarch/memset-vec-unaligned-erms.S
+++ b/sysdeps/x86_64/multiarch/memset-vec-unaligned-erms.S
@@ -114,6 +114,7 @@
 # error SECTION is not defined!
 #endif
 
+	.section SECTION(.text), "ax", @progbits
 #if IS_IN (libc)
 # if defined SHARED
 ENTRY_CHK (WMEMSET_CHK_SYMBOL (__wmemset_chk, unaligned))
